suite/tests/client-interface/drmgr-test.c

```diff
@@ -315,6 +315,20 @@ main(int argc, char **argv)
     print("%c\n", table[1]);
 #    endif
 
+#    ifdef X86
+    /* Test xsave for drutil_opnd_mem_size_in_bytes. We're assuming that
+     * xsave support is available and enabled, which should be the case
+     * on all machines we're running on.
+     * XXX i#2946: support Windows.
```

Comment:
Windows is easier than Linux: just `_xsave(buffer,  -1)`

Reply:
Done (though need the Windows equivalent alignment syntax :) not sure)

Reply:
Most such questions are quickly answered by just searching our git repo:
```C
core/lib/globals_shared.h:#    define ALIGN_VAR(x) __declspec(align(x))
core/lib/globals_shared.h:#    define ALIGN_VAR(x) __attribute__((aligned(x)))
```